test/integration/scheduler_perf/dra/performance-config.yaml

```diff
@@ -507,3 +507,85 @@
       measurePods: 2500
       measureClaims: 500 # must be measurePods / 5
       maxClaimsPerNode: 2
+
+# DRABindingConditionsTest validates DRA device binding condition handling mechanisms.
+# - Success scenario: DeviceAttached=True → Pod should schedule
+# - Failure scenario: AttachmentFailed=True → Pod should NOT schedule
+- name: DRABindingConditionsTest
+  featureGates:
+    DynamicResourceAllocation: true
+    DRADeviceBindingConditions: true
+    DRAResourceClaimDeviceStatus: true
+  workloadTemplate:
+    - opcode: createNodes
+      nodeTemplatePath: templates/node-with-dra-test-driver.yaml
+      countParam: $nodesWithDRA
+    - opcode: createResourceDriver
+      driverName: test-driver.cdi.k8s.io
+      nodes: scheduler-perf-dra-*
+      maxClaimsPerNodeParam: $maxClaimsPerNode
+    - opcode: createAny
+      templatePath: templates/deviceclass.yaml
+    - opcode: createAny
+      namespace: test
+      templatePath: templates/resourceclaimtemplate.yaml
+    - opcode: createPods
+      namespace: test
+      podTemplatePath: templates/pod-with-claim-template.yaml
+      countParam: $measurePods
+      steadyState: true
+      collectMetrics: true
+    - opcode: updateDeviceConditions
+      namespace: test
+      ConditionTypeParam: $conditionType
+      ConditionStatusParam: $conditionStatus
+      durationParam: $updateDuration
+    - opcode: checkPodScheduled
+      namespace: test
+      ScheduledParam: $scheduled
+      durationParam: $checkDuration
+  workloads:
+    - name: fast_AttachmentSuccess
+      labels: [integration-test, short]
+      params:
+        nodesWithDRA: 1
+        measurePods: 1
+        maxClaimsPerNode: 1
+        conditionType: DeviceAttached
+        conditionStatus: "True"
+        scheduled: true
+        updateDuration: 30s
+        checkDuration: 30s
+    - name: fast_AttachmentFailed
+      labels: [integration-test, short]
+      params:
+        nodesWithDRA: 1
+        measurePods: 1
+        maxClaimsPerNode: 1
+        conditionType: AttachmentFailed
+        conditionStatus: "True"
+        scheduled: false
+        updateDuration: 30s
+        checkDuration: 30s
+    - name: multiple_pods_AttachmentSuccess
```

Comment:
This is multiple pods each with their own claim. This is worthwhile, but please also add scenarios for multiple pods sharing the same claim. That leads to those interesting corner cases discussed in the KEP.